mkdocs.yml

```diff
@@ -42,6 +42,7 @@ extra_css:
 
 extra_javascript:
     - javascript/javadoc_menu_dropdown.js
+    - javascript/copy_maven_command.js
```

Comment:
@Nemikor Can you just change this file to be something like copy_to_clipboard.js?